debian/libmercury1.install

```diff
@@ -1 +1,2 @@
+usr/bin/*
```

Comment:
This should go into a `mercury.dirs` file.  The `libmercury1` package is for the shared library only.